admin_guide/managing_networking.adoc

```diff
@@ -1109,9 +1108,14 @@ xref:../admin_guide/managing_projects.adoc#modifying-the-template-for-new-projec
 $ oc edit template project-request -n default
 ----
 +
-Add each default policy as a new element in the `objects` array. Eg:
-+
+[NOTE]
+====
+To include `NetworkPolicy` objects into existing template, use the `oc edit` command. Currently, it is not possible to use `oc patch` to modify a `Template` object.
```

Comment:
I realized after submitting the PR that it would be more accurate to say

    it is not possible to use `oc patch` to add objects to a `Template` resource.

(ie, you *can* use `oc patch` to modify a Template, you just have to specify the entire new value of the `objects` array rather than only the new item, so it's not very useful.)